src/components/common/helpers/Helpers.tsx

```diff
@@ -880,3 +880,21 @@ export function useSearchString(): UseSearchString {
 
     return { queryParams, searchParams }
 }
+
+export const sortOptionsByLabel = (optionA, optionB) => {
```

Comment:
something similar is already there in utils may be. just check and see if we can utilize that.

Reply:
Yes. Tried using that util but there's some type mis-match issue so as discussed going ahead with these functions.